File: DS&DAA-using-python/insertionsort.py
#taking array input from user
n = int(input("Enter the no of ekement in unsoterd array\n"))
arr = []
print("Enter the elements of array")
for i in range(n):
    arr.append(int(input()))

#printing unsorted array
print("Unsorted array\n",arr)

#main stetment where shorting perform
for i in range(1,n):
    value = arr[i]
    hole = i

    while(hole>0 and arr[hole-1]>value):
        arr[hole] = arr[hole-1]
        hole = hole-1

    arr[hole] = value

#printing unsorted array
print("Sorted array\n",arr)    

File: DS&DAA-using-c/insertionsort.c
#include<stdio.h>

void main(){
    int arr[50],n,i,j,temp;
    /*Taking user input any array*/
    printf("Enter the no of element in array");
    scanf("%d",&n);
    printf("Enter the element of array");
    for(i=0;i<n;i++){
        scanf("%d",&arr[i]);       
    }
    /*Printing unshorted array*/
    printf("Unshorted array\n");
    for(i=0;i<n;i++){
        printf("%d",arr[i]);
    }